test/integration/create-cluster-on-hub.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function token_cleanup() {
+	curl -X DELETE https://discovery-stage.hub.docker.com/v1/clusters/$1
```

Comment:
We should probably wait for a `swarm destroy` command so you can actually test the creation and then the destruction of a cluster.